### DISCUSSION TITLE
The comprehensive of Python and its future?

### DISCUSSION THREAD

**Discussion by branlabs** [python/typing#1277] [General]
Python is a programming language that can be used in many areas such as AI, creating a Web Server, and writing tools that run in many industries (DevOps, Hacking, etc.). So is Python comprehensive and what does its future look like?

**Comment by jeremyn** [comment]
@nguyenlethaihoang This post isn't really on-topic for this forum, please see https://github.com/python/typing/discussions/826. (I've asked @srittau to pin that discussion to make it more visible.)
